gnomad/resources/grch37/reference_data.py

```diff
@@ -287,3 +335,18 @@ def get_truth_ht() -> hl.Table:
         .repartition(200, shuffle=False)
         .persist()
     )
+
+
+gtex_rsem = VersionedTableResource(
+    default_version="v7",
+    versions={
+        "v7": GnomadPublicTableResource(
+            path="gs://gnomad-public-requester-pays/resources/grch38/gtex_rsem/gtex_rsem_v7.mt",
+            import_func=_import_gtex_rsem,
+            import_args={  # TODO: update path
+                "gtex_path": "gs://gnomad-qin/bulk-gex_v7_rna-seq_GTEx_Analysis_2016-01-15_v7_RSEMv1.2.22_transcript_tpm.txt.gz",
+                "meta_path": "gs://gnomad-qin/annotations_v7_GTEx_v7_Annotations_SampleAttributesDS.txt.gz",
+            },
```

Comment:
We might end up using the same import function for the grch38 resource, so I think we should have this function take in other possible import parameters.

```suggestion
                "min_partitions": 1000,
                 "reference_genome": "GRCh37",
            },
```